src/components/FlowRunsBarChart.vue

```diff
@@ -163,9 +160,10 @@
 <style>
 .flow-runs-bar-chart { @apply
   relative
-  grid
-  items-end
-  grid-rows-1;
+  w-full
+  h-full
+  flex
```

Comment:
I switched this to flex to remove the need to calculate columns, plus in a situation where for some reason more bars are trying to render than the width allows then the bars won't start wrapping, they'll just overflow to the side.